src/status_im2/contexts/syncing/scan_sync_code/view.cljs

```diff
@@ -214,11 +214,15 @@
     [:<>
      [rn/view {:style style/camera-container}
       [camera-kit/camera
-       {:ref            #(reset! camera-ref %)
-        :style          style/camera-style
-        :camera-options {:zoomMode :off}
-        :scan-barcode   true
-        :on-read-code   on-read-code}]]
+       {:ref          #(reset! camera-ref %)
+        :style        style/camera-style
+        :camera-type  camera-kit/camera-type-back
+        :zoom-mode    :off
+        ;; TODO: Add flash toggle
+        ;; https://github.com/status-im/status-mobile/issues/16243
```

Comment:
Since the GH issue is well defined, I think it's fine to delete the TODO comment.